src/main/java/sirius/biz/mongo/MongoBizEntity.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.mongo;
+
+import sirius.biz.protocol.NoJournal;
+import sirius.biz.protocol.TraceData;
+import sirius.biz.protocol.Traced;
+import sirius.db.jdbc.SQLEntity;
+import sirius.db.mixing.Mapping;
+import sirius.db.mongo.MongoEntity;
+
+/**
+ * Provides a base class for entities managed by a {@link sirius.biz.web.BizController}.
+ * <p>
+ * Provides built in {@link TraceData}
+ */
+public abstract class MongoBizEntity extends MongoEntity implements Traced {
+
+    /**
+     * Contains tracing data which records which user created and last edited the entity
+     */
+    public static final Mapping TRACE = Mapping.named("trace");
+    private final TraceData trace = new TraceData();
+
+    @Override
+    public TraceData getTrace() {
+        return trace;
+    }
+
+    /**
+     * Checks whether any {@link Mapping} (except {@link NoJournal no journal data}) of the current {@link MongoBizEntity} changed.
+     *
+     * @return <tt>true</tt> if at least one column was changed, <tt>false</tt> otherwise.
+     */
+    public boolean isAnyColumnChangedExceptNoJournal() {
```

Comment:
Doens't seem to be implementation specific. can we move this to a parent class for every db-implementation?

Reply:
difficult, as their base classes meet in sirius-db where NoJournal is undefined - the only other option would be an interface with default method - as we do not expect many more of those entity classes, two methods isn't that bad anyway..